inject/chrome-mv3.js

```diff
@@ -0,0 +1,36 @@
+/* global contentScopeFeatures */
+
+const secret = window.crypto.randomUUID()
+
+contentScopeFeatures.load()
+
+window.addEventListener(secret, ({ detail: message }) => {
+    switch (message?.type) {
```

Comment:
Nit: Rather than all the ? checking for message an early return would make the code simpler.
```suggestion
    if (message) {
        return
    }
    switch (message.type) {
```

Also typescript is a little dumb sometimes I'm guessing `contentScopeFeatures.init(message?.argumentsObject)` was added because TS was complaining; but you're already returning early in that switch 😢

Reply:
Done.